woocommerce/woocommerce-android
# Issue/5811 order creation dictate quantity change

Closes #5811 - prior to this PR, tapping the plus or minus buttons next to a product in order creation wouldn't dictate the change. This PR resolves this by relying on [android:accessibilityLiveRegion](http://androidsbs.blogspot.com/2013/11/androidaccessibilityliveregion.html) to announce changes to the product quantity.

To test:

* Enable TalkBack
* Go to create an order
* Add a product to the order
* Notice that when the plus or minus buttons are tapped, the change in quantity is spoken
